addons/web/static/src/views/view_compiler.js

```diff
@@ -425,8 +425,9 @@ export class ViewCompiler {
      * @returns {Element}
      */
     compileWidget(el) {
+        const widgetId = el.getAttribute("widget_id");
         const attrs = {};
-        const props = { record: "__comp__.props.record", readonly: this.ctx.readonly };
+        const props = { record: "__comp__.props.record" };
         for (const { name, value } of el.attributes) {
```

Comment:
for loop can probably be replaced by two ifs (name and value), no need to iterate on all attrs anymore